### PR TITLE
Add help message when user has no notifications

### LINKED ISSUE #2563: UI if no notifications are set up
As a platform operator, if i have no notifications set up, I will see a pop up message informing me of some providers and linking me to the notification controller documentation so that I can learn how to configure notifications.

Microcopy: 
Heading - No notifications are currently configured
Body -
Set up notifications to alert on changes via multiple different platforms such as Slack, Azure Event Hub, Grafana, OpsGenie and many more!

To learn more about how to set up notifications, visit our documentation [link to docs](https://fluxcd.io/flux/guides/notifications/)

 Figma Link https://www.figma.com/file/IVHnM9iyeFWpd11evtY8ux/Weave-GitOps?node-id=4908%3A9125

20220829No notifications set up

### PR DESCRIPTION
**Pull request by joshri**
Closes #2563 

If the ListNotifications request is a success but returns no notifications, a nice little helpful message will be displayed with a link to the flux docs:
image

### CODE REVIEW

**Comment by opudrovs** [comment]
I THINK the vertical space between message text and heading can be smaller. It might be better to decrease it and add or increase the space at the bottom if you feel the message is not toll enough. Or you could just decrease the space between the message text and heading and center all text vertically in the message box. Right now this large gap between lines of text looks a bit strange.

**Comment by opudrovs** [comment]
BTW, if using the Spacer component is preferred to setting the margins manually, I am totally using this component in the next graph PR to rework the No Dependencies message.

**Comment by joshri** [comment]
> #2563



> #2563

I tried to make it as similar to the screenshot from the issue as I could - going to attempt to link it here 
https://user-images.githubusercontent.com/92439/187241214-19468096-533f-4502-9329-4204e732163e.png

**Comment by opudrovs** [comment]
@joshri I see, OK, then, still the gap looks weird but if it's what product wants let it be.

File: ui/components/NotificationsTable.tsx [hunk 1]
```diff
@@ -93,6 +96,32 @@ function NotificationsTable({ className, rows }: Props) {
...
+          <Text>
+            Set up notifications to alert on changes via multiple different
+            platforms such as Slack, Azure Event Hub, Grafana, OpsGenie and many
+            more!
+          </Text>
+          <Spacer padding="xs" />
+          <Text>
+            To learn more about how to set up notifications,
+            <Link href="https://fluxcd.io/flux/guides/notifications/" newTab>
+              {" "}
```
[the remark this answers is not in the document]
Comment (joshri) [reply ui/components/NotificationsTable.tsx:117]: Didn't know about HTML entities - that was a good read!

**Comment by joshri** [comment]
> BTW, if using the Spacer component is preferred to setting the margins manually, I am totally using this component in the next graph PR to rework the No Dependencies message.

Like most things in our code it's not amazing semantically - but I definitely prefer it to a lot of styling lines / inline styles / an extra styled component

File: ui/components/NotificationsTable.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (opudrovs) [reply ui/components/NotificationsTable.tsx]: Yes, they can be very useful.

**Review by opudrovs — APPROVED** [review]

**Pull request merged**